text-serializer-gson/src/main/java/net/kyori/adventure/text/serializer/gson/GsonComponentSerializer.java

```diff
@@ -111,20 +110,21 @@ static Builder builder() {
    */
   @NotNull JsonElement serializeToTree(final @NotNull Component component);
 
+  /**
+   * Creates a new {@link Builder} from this {@link GsonComponentSerializer}.
+   *
+   * @return a new {@link Builder}
+   * @since 4.9.0
+   */
+  @Override
+  @NotNull GsonComponentSerializer.Builder toBuilder();
+
   /**
    * A builder for {@link GsonComponentSerializer}.
    *
    * @since 4.0.0
    */
-  interface Builder extends Buildable.Builder<GsonComponentSerializer> {
-    /**
-     * Sets that the serializer should downsample hex colors to named colors.
-     *
-     * @return this builder
-     * @since 4.0.0
-     */
-    @NotNull Builder downsampleColors();
```

Comment:
I don't think we can remove these methods from this builder as this is a breaking change I _thinkkkk_